docs/config-api.md

```diff
@@ -175,6 +175,13 @@ System.config({
 });
 ```
 
+* `format`: Overrides in what format the module is loaded. See [Module Formats](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md).
+* `exports`: Used to create a global variable, pointing to this meta path. Only available at the time of executing the module referenced by meta path. When meta is used inside `packages` configuration this variable is only available within that package. More info: [Exports](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md#exports)
```

Comment:
Alright, I understand now it's incorrect, but your description is very short and for me personal unclear. For example is the global really global? Or can it be limited to a package by using it within a packages config? 
What about:
> For `global` format, when automatic detection of exports is not enough, a custom exports meta value can be set. More info: [Exports](https://github.com/systemjs/systemjs/blob/master/docs/module-formats.md#exports)